examples/app_commands/slash_options.py

```diff
@@ -29,5 +30,18 @@ async def channel(
     await ctx.respond(f"Hi! You selected {channel.mention} channel.")
 
 
+@bot.slash_command(name="attach_file")
+async def say(
+    ctx: discord.ApplicationContext,
+    attachment: Option(
+        discord.Attachment,
+        "File to attach to the message?",

```

Comment:
This could be better maybe ```suggestion
        "The file to attach to the message",
```